release-notes/major12/12.0.0/valtimo-frontend-libraries.md

```diff
@@ -4,15 +4,15 @@
 
 The following features were added:
 
-* **New feature1**
+* **Inline edit document definition**
 
-  Description of the new feature goes here.
-  Also link to the page that expains the feature in greater detail.
+  Document definition can now be edited inline to create a new version.
 
-* **New feature2**
+* **Pending changes modal extension**
 
-  Description of the new feature goes here.
-  Also link to the page that expains the feature in greater detail.
+  The pending changes modal has been extended to accept custom modals. An example of this use would be a pending changes modal for an 'Edit mode' function, where you might want to have three buttons(e.g. *Keep editing*, *Save*, *Discard*) instead of the out of the box two.
```

Comment:
```suggestion
  The pending changes modal has been extended to accept custom modals. An example of this use would be a pending changes modal for an 'Edit mode' function, where you might want to have three buttons (e.g. *Keep editing*, *Save*, *Discard*) instead of the two buttons that are offered out of the box.
```